### PR TITLE
RGB spaces

### PR DESCRIPTION
**Pull request by Crissov**
- adds dci-p3
- some matrix/array number formatting (trailing zeros)
- adds some initial jsdoc documentation
- uses α, β, γ, Γ, φ, K₀ more systematically

### CODE REVIEW

**Comment by svgeesus** [comment]
All this in one PR, @Crissov ? Please split into separate ones, and rebase.

**Comment by LeaVerou** [comment]
Also please note that #100 also adds jsdoc, hopefully the jsdoc this one adds is different.

Thanks for contributing!

File: src/spaces/srgb.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
 import Color, {util} from "./../color.js";
 
+/**
+ * sRGB
+ *
+ * @see https://webstore.iec.ch/publication/6169 IEC 61966-2-1:1996
+ * @see https://www.w3.org/Graphics/Color/sRGB.html
```
Comment (svgeesus) [inline src/spaces/srgb.js:7]: Don't cite that one, it is the historical older proposal

File: src/CATs.js [hunk 2]
```diff
@@ -91,9 +91,9 @@ Color.defineCAT({
 Color.defineCAT({
 	id: "CAT16",
 	toCone_M: [
-		[  0.401288,  0.650173, -0.051461 ],
-		[ -0.250268,  1.204414,  0.045854 ],
-		[ -0.002079,  0.048952,  0.953127 ]
+		[  0.4012880,  0.6501730, -0.0514610 ],
```
Comment (svgeesus) [inline src/CATs.js:94]: In general I agree about making clear the number of significant figures and thus, not dropping trailing zeroes.

File: src/spaces/dci-p3.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * @see https://doi.org/10.5594%2FSMPTE.EG432-1.2010 SMPTE EG 432-1 
+ * @see https://doi.org/10.5594%2FSMPTE.RP431-2.2011 SMPTE RP 431-2
+ */
+Color.defineSpace({
+	inherits: "srgb",
+	id: "dci-p3",
+	name: "DCI P3",
+	cssId: "dci-p3",
+	white: Color.whites.Theater,
```
Comment (svgeesus) [inline src/spaces/dci-p3.js:15]: Where is this added? Should be in CATS.js

File: src/spaces/dci-p3.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+	// Gamma correction is the same as sRGB
+	// convert an array of dci-p3 values to CIE XYZ
+	// using Theater {0.3140, 0.3510} instead of D65 = {0.3127, 0.3290} (no chromatic adaptation)
+	// http://www.brucelindbloom.com/index.html?Eqn_RGB_XYZ_Matrix.html
+	// Functions are the same as sRGB, just with different matrices
+	toXYZ_M: [
```
Comment (svgeesus) [inline src/spaces/dci-p3.js:21]: Are these calculated directly from primaries and whites, or copied from Lindbloom?

File: src/spaces/prophoto.js [hunk 5]
```diff
@@ -1,43 +1,50 @@
...
+ * @see https://webstore.ansi.org/Standards/I3A/ANSII3AIT1076662002 ANSI/I3A IT10.7666-2002
+ * @see http://www.color.org/ROMMRGB.pdf
+ */
 Color.defineSpace({
 	inherits: "srgb",
 	id: "prophoto",
 	name: "ProPhoto",
 	cssId: "prophoto-rgb",
 	white: Color.whites.D50,
+	α /*= a + 1*/: 1.0,
```
Comment (svgeesus) [inline src/spaces/prophoto.js:18]: Where do these constants come from? Did you buy the standards?

File: src/spaces/rec2020.js [hunk 6]
```diff
@@ -1,46 +1,49 @@
...
 	id: "rec2020",
 	name: "REC.2020",
-	α: 1.09929682680944,
-	β: 0.018053968510807,
-	toLinear(RGB) {
-		const {α, β} = this;
+	α: 1.09929682680944,  // will often be assumed to be 1.099 for 10-bit systems and 1.0993 for 12-bit systems
+	a: 0.09929682680944,
+	β: 0.018053968510807, // will often be assumed to be 0.018 for 10-bit systems and 0.0181 for 12-bit systems
+	γ /* > 1 */: 12/5 /* = 2.4 */,
```
Comment (svgeesus) [inline src/spaces/rec2020.js:17]: Where do these come from?

**Review by svgeesus — APPROVED** [review]
Some nice stuff, some puzzling, but would prefer to see smaller PRs each of which does one thing

File: src/spaces/dci-p3.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
+import Color from "./srgb.js";
+
+Color.defineSpace({
+	inherits: "srgb",
+	id: "dic-p3",
```
Comment (svgeesus) [inline src/spaces/dci-p3.js]: dci not dic  :)

**Review by svgeesus — CHANGES_REQUESTED** [review]
Please split into separate ones, and rebase.

**Comment by Crissov** [comment]
This PR was clearly premature. I had a weak moment today when submitting it, after I had rediscovered that I had been working on some improvements quite a while ago.

I hope I can fix some of it into smaller PRs and address your other comments as soon as tomorrow.

File: src/spaces/prophoto.js (hunk 5 above)
Comment (Crissov) [reply src/spaces/prophoto.js:18]: My alma mater has full ISO and DIN subscriptions, but I should check again since I wrote this part a while ago.

**Pull request closed without merging**